.eslintrc.js

```diff
@@ -1 +1,10 @@
-module.exports = require('./packages/scripts/eslint-preset')
```

Comment:
suggested by @mattpocock and I like this approach because it allows for more local configuration